Sources/DiffableDataSourceSnapshot.swift

```diff
@@ -1,28 +1,71 @@
+import UIKit
```

Comment:
should this change also works on AppKit? 🤔

Reply:
Yeah that would make sense to import either UIKit/AppKit conditionally. As [NSDiffableDataSourceSnapshot](https://developer.apple.com/documentation/appkit/nsdiffabledatasourcesnapshot) seems compatible with both.